model/config/config.yaml

```diff
@@ -13,10 +13,50 @@ logger:
         handlers: [console]
 
 IGPreprocessor:
-    raw_data_path: ./.cache/data/instagram/ig_sample.tar.gz
-    raw_data_url: https://tweakstory.stefanmesken.info/data/ig_sample.tar.gz
-    raw_data_directory: ig_sample
+    raw_data_path: "./.cache/data/instagram/ig_sample.tar.gz"
+    raw_data_url: "https://tweakstory.stefanmesken.info/data/ig_sample.tar.gz"
+    raw_data_directory: "ig_sample"
     raw_data_group_names:
-        insta-caption-train.json: train
-        insta-caption-test1.json: eval
-        insta-caption-test2.json: test 
+        insta-caption-train.json: "train"
+        insta-caption-test1.json: "eval"
+        insta-caption-test2.json: "test"
+
+Trainer:
+    pad_token_id: 0
+    unk_token_id: 0
```

Comment:
why UNK and PAD has the same index?
Also, when I look back the pretrained word embedding layer in `AttentionLSTMDecoder`, it seems there is no enforcement as to align these index to the pretrained word embedding's. 
(e.g `eos_token_id` is never used in `AttentionLSTMDecoder`, `bos_token_id` is not applied to the pretrained embedding)

Reply:
> why UNK and PAD has the same index?

This pretrained embedding doesn't come with a <pad> token, so I'm using its <unk> token instead. That's fine because I ignore it in the loss function.

> it seems there is no enforcement

That's taken care of in the preprocessor. See https://github.com/riven314/TweakStory/blob/8ed4cb01125d239e8a534fecd9c146136cf7835d/model/preprocessor.py#L482